<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProduct</class>
 <widget class="QWidget" name="CreateProduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Imagenes/tweety2.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_cod">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>167</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>código</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cod">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>73</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Código:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_cant">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>79</y>
     <width>167</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>cantidad</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_create">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>301</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Crear</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_descrip">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>101</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Descripción:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>1</y>
     <width>300</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Crear Nuevo Producto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_categoria">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>182</y>
     <width>86</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Categoría:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>301</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_precio">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>151</y>
     <width>57</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Precio:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_categoria">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>172</y>
     <width>167</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>categoría</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>301</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_descrip">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>167</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>nombre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_precio">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>141</y>
     <width>167</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>precio</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cant">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>89</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cantidad:</string>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>input_cant</zorder>
  <zorder>btn_create</zorder>
  <zorder>label_descrip</zorder>
  <zorder>input_cod</zorder>
  <zorder>label</zorder>
  <zorder>label_categoria</zorder>
  <zorder>btn_exit</zorder>
  <zorder>label_precio</zorder>
  <zorder>input_categoria</zorder>
  <zorder>btn_cancel</zorder>
  <zorder>input_descrip</zorder>
  <zorder>label_cod</zorder>
  <zorder>input_precio</zorder>
  <zorder>label_cant</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
